packages/connectivity/src/scp-cf/destination/destination-accessor.ts

```diff
@@ -85,3 +102,79 @@ export async function getDestination(
     (await getDestinationFromDestinationService(options));
   return destination;
 }
+
+/**
+ * Creates comprehensive log messages from a destinations array and their origin.
+ * @param origin - Origin of the destination.
+ * @param destinations - Array of destinations.
+ * @returns Logs of the retrival of destinations.
+ */
+function destinationFetchLogHelper(
```

Comment:
[pp] the `helper`feels a bit strange and our style guide also says: user verbs (https://github.com/SAP/cloud-sdk-js/blob/main/STYLEGUIDE.md).